extras/travis_build.sh

```diff
@@ -35,6 +35,8 @@ set -e
 if [[ $OSTYPE = darwin* ]]; then
     export CC=$(ls /usr/local/bin/gcc-* | grep '^/usr/local/bin/gcc-\d$')
     export CXX=$(ls /usr/local/bin/g++-* | grep '^/usr/local/bin/g++-\d$')
+    #Ensure that nosetests path can be found
+    export PATH=/usr/local/bin/nosetests:$PATH
```

Comment:
Adjust to user-local installation:
```suggestion
    export PATH=$HOME/.local/bin:$PATH
```